firebase-installations-interop/src/main/java/com/google/firebase/installations/InstallationTokenResult.java

```diff
@@ -21,30 +21,44 @@
 @AutoValue
 public abstract class InstallationTokenResult {
 
-  /** A new FIS Auth-Token, created for this Firebase Installation. */
+  /**
+   * A new FIS Auth-Token, created for this Firebase Installation.
+   *
+   * @hide
+   */
   @NonNull
   public abstract String getToken();
   /**
    * The amount of time, in seconds, before the auth-token expires for this Firebase Installation.
+   *
+   * @hide
    */
   @NonNull
   public abstract long getTokenExpirationTimestamp();
```

Comment:
This one was part of API review as well, although there you called it `getExpirationTimestamp()`.

Overall I'd suggest you go through all API's and compare them with the API review, thanks.

Reply:
for `getExpirationTimestamp()` will update the  AP review doc. Updated the FIS SDK code `getToken(boolean)` as per the api review.